### PR TITLE
Explicitly set pyBullet physicsClientId everywhere

### PR DESCRIPTION
**Pull request by TheInfamousWayne**
created client_id for individual envs

### CODE REVIEW

**Comment by luator** [comment]
@TheInfamousWayne I noticed that the client id was not set for the CubeMarker.  I added a commit, please verify that this is correct.

File: example/test.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
```
Comment (luator) [inline example/test.py:1]: I think this script should not be added here

**Review by luator — CHANGES_REQUESTED** [review]
Looks good to me.  Only the test.py should probably be removed.

**Comment by luator** [comment]
I created a "develop" branch and changed this PR to merge there.  This way we can merge the different PRs there and test together before pushing it to master.

File: example/test.py (hunk 1 above)
Comment (shrutij01) [reply example/test.py:1]: +1

**Review by shrutij01 — APPROVED** [review]
looks great to me, thanks! :metal:

**Review by luator — APPROVED** [review]

**Comment by luator** [comment]
@TheInfamousWayne I deleted test.py so I can already merge for testing.  Let me know in case there is a problem with the changes I did, then we can fix it in a new PR.

**Pull request merged**